source/ed/build_helper.cpp

```diff
@@ -840,7 +840,6 @@ void builder::finish() {
     generate_dummy_basis();
     data->pt_data->sort_and_index();
     data->build_uri();
-    data->build_raptor();
```

Comment:
Why you suppress build_raptor() inside the builder make function. 
It useful to have all methods call inside. No ?

Reply:
It's already called in finish() above: https://github.com/CanalTP/navitia/blob/dev/source/ed/build_helper.cpp#L836
I explained it in the commit message (if you think I should mention it elsewhere).

Reply:
Oups my bad